src/components/Modal/index.js

```diff
@@ -25,16 +26,57 @@ const defaultLoadingContent = (
 );
 
 const ModalHeader = ({ theme, title, unsetModal, ...props }) => (
-  <h2
+  <div
     css={`
-      ${theme.profileH2} ${theme.row} justify-content: space-between;
+      ${theme.row} justify-content: space-between;
+      border-bottom: 1px solid #d4d6dd;
+      margin-bottom: 1.5em;
     `}
   >
-    <span>{title}</span>
-    <CloseIcon css="cursor:pointer;" onClick={() => unsetModal()} />
-  </h2>
+    <span css={theme.modalTitle}>{title}</span>
```

Comment:
We should probably try to add class names along side styling so Kim can inspect more easily :)